### PR TITLE
fix: fix export jpg format error(v8)

### PR DESCRIPTION
**Pull request by yungzhu**
rt

### CODE REVIEW

**Comment by yungzhu** [comment]
image

**Comment by Zyie** [comment]
Hey @yungzhu could you update this PR to the latest `next-v8`? 
There seems to be some weird merge issues

File: src/rendering/renderers/shared/extract/ExtractSystem.ts [hunk 1]
```diff
@@ -98,6 +98,8 @@ export class ExtractSystem implements System
 
         const { format, quality } = options;
 
+        const imageFormat = format === 'png' ? 'image/png' : 'image/jpeg';
```
Comment (bigtimebuddy) [inline src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: I don't think this is correct. The format option should be mime type not "png". Can you please create a bug for this?

**Review by bigtimebuddy — CHANGES_REQUESTED** [review]

File: src/rendering/renderers/shared/extract/ExtractSystem.ts (hunk 1 above)
Comment (yungzhu) [reply src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: image
Actually the argument that's passed in is this

Comment (bigtimebuddy) [reply src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: The fix here should probably be this instead:

```ts
type Formats = 'image/png' | 'image/jpeg';
```

Comment (yungzhu) [reply src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: That's fine too, but png and jpg are easier to use

Comment (bigtimebuddy) [reply src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: If that's the case, then probably a map:
```ts
const imageTypes: Record<Formats, string> = {
  png: 'image/png',
  jpg: 'image/jpeg',
  webp: 'image/webp',
};
```
Personally, I think the mime type is more transparent, though slightly more verbose, with how toDataURL and other canvas methods are invoked.

Comment (yungzhu) [reply src/rendering/renderers/shared/extract/ExtractSystem.ts:101]: I agree. In fact, I can accept all of them

**Pull request closed without merging**